### PR TITLE
Migrate to Tool Kit

### PR DESCRIPTION
**Pull request by rowanmanning**
🚀🧰🎄

### CODE REVIEW

File: .toolkitrc.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+plugins:
+  - "@dotcom-tool-kit/component"
+  - "@dotcom-tool-kit/eslint"
+hooks: {}
```
Comment (apaleslimghost) [inline .toolkitrc.yml:4]: **nitpick**: an empty `hooks` is redundant, you can remove this (i know it was generated by the migration script! we should probably have it not output empty things?)

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.circleci
```
Comment (apaleslimghost) [inline .npmignore:1]: **thought**: lol were we publishing this before 😅

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1 @@
-.editorconfig
-.eslintrc.js
-/node_modules/
+node_modules
```
Comment (apaleslimghost) [inline .gitignore:1]: **chore**: you should also ignore the `.toolkitstate` directory (the most recent version of Tool Kit prompts you to do this in the migration script, i guess this was run with a slightly old version?)

**Review by apaleslimghost — APPROVED** [review]

File: .npmignore (hunk 2 above)
Comment (rowanmanning) [reply .npmignore:1]: yeah just everything

**Pull request merged**